src/main/java/duke/commands/DoneCommand.java

```diff
@@ -0,0 +1,42 @@
+package duke.commands;
+
+import duke.items.Item;
+import duke.ItemList;
+import duke.Ui;
+import java.util.ArrayList;

```

Comment:
Wrong Import Statement order (should be lexicographical and 
Custom imports should be after Java Standard imports)
```suggestion
import java.util.ArrayList;

import duke.ItemList;
import duke.Ui;
import duke.items.Item;
```